back/app/services/invites/service.rb

```diff
@@ -99,7 +99,8 @@ def build_invitee(params, default_params = {})
   def prepare_invitee(params, default_params)
     email = params['email']&.strip
     group_ids = params['group_ids'] || default_params['group_ids'] || []
-    roles = params['roles'] || default_params['roles'] || []
+    default_params_roles = default_params['roles'].presence || []
+    roles = (params['roles'] + default_params_roles).uniq
```

Comment:
As it solves the bug that we found on Friday, it would be useful to add the ticket number to the commit message (maybe with `--amend`) to properly log why and when we've changed it.